openfasoc/common/platforms/sky130hvl/fastroute.tcl

```diff
@@ -1,4 +1,5 @@
-set_global_routing_layer_adjustment $::env(MIN_ROUTING_LAYER)-$::env(MAX_ROUTING_LAYER) 0.5
+# reduces the routing resources of all routing layers by 30%
+set_global_routing_layer_adjustment * 0.3
```

Comment:
What is this added?

Reply:
While using earlier command the adjustment setting was not changed in the flow. The updated command will change the setting for metal1 to metal5 .

Reply:
Why are you reducing routing resources by 30% 
`set_global_routing_layer_adjustment * 0.3`

Reply:
This is a default setting that is been used by OpenROAD-flow-scripts.